README.md

```diff
@@ -63,34 +63,50 @@ run the entire CMR with no external dependencies from this Jar file and use
 that instance for local testing. The sections below contain instructions for
 running the CMR as a single process or as multiple processes.
 
-#### First Steps
-
-1. Ensure you have installed on your system the items listed above in the
-   "Prerequisites" section.
-1. Download the Oracle JDBC `.jar` files into `./oracle-libs/support` by
-   following instructions in `./oracle-lib/README.md`. (The CMR must have these
-   libraries to build but it does not depend on Oracle DB when running
-   locally. It uses a local in-memory database by default.)
-
 #### Using the `cmr` CLI Tool
 
 This project has its own tool that is used for everything from initial setup to
-running buids and tests on the CI/CD infrastructure. In order to use the tool
+running builds and tests on the CI/CD infrastructure. In order to use the tool
 as we do below, be sure to run the following from the top-level CMR directory:
 
 ```
 export PATH=$PATH:`pwd`/bin
 source resources/shell/cmr-bash-autocomplete
 ```
 
-If you use another system shell, we'll accept a PR with auto-complete for it.
+(If you use a system shell not compatible with Bash, we'll accept a PR with
+auto-complete for it.)
+
+To make this change permanent:
+
+```
+echo "export PATH=\$PATH:`pwd`/bin" >> ~/.profile
+echo "source `pwd`/resources/shell/cmr-bash-autocomplete" >> ~/.profile
+```
+
+#### Oracle Dependencies
+
+Even if you're not going to develop against a local Oracle database,
+you still need to have the Oracle libraries locally installed to use the
+CMR.
+
+Here are the steps to do so:
+
+1. Ensure you have installed on your system the items listed above in the
+   "Prerequisites" section.
+1. Download the Oracle JDBC JAR files into `./oracle-libs/support` by
```

Comment:
These step number should increment, e.g. this should be `2`, the next `3`, etc. This applies to other parts of the file that has steps as well.

Reply:
This is a feature of Markdown -- it sees the `1.` and turns the whole thing into a properly numbered list. This removes the hassle of re-numbering when you change the order, remove, or add items.

Reply:
oh. I didn't know that. Thanks!